authors/apps/articles/tests/test_favorite_articles.py

```diff
@@ -0,0 +1,57 @@
+from .base_test import TestBaseCase
+
+from rest_framework import status
+from .test_like_dislike_articles import TestLikeDislike
+
+
+class TestFavoriteArticles(TestBaseCase):
+    def base_create_article(self, token):
+        """
+        creates an articles
+        """
+        url = self.create_list_article_url
+        response = self.client.post(url, self.article, format='json',
+                                    HTTP_AUTHORIZATION='Token ' + token)
+        return response
+
+    def favorite_Article(self, url, token):
+        """
+        allows a logged in user to favorite article
+        """
+        response = self.client.put(
+            url,
+            format='json',
+            HTTP_AUTHORIZATION='Token ' + token
+        )
+        return response
+
+    def test_favorite_Article(self):
+        token = self.login_user()
+        url = self.create_list_article_url
```

Comment:
@Nicanor008 line 30 - 33 can be replaced by just calling slug = self.create_article() from our base test

Reply:
Checking this out @eduhmik

Reply:
Feedback implemented